internal/dispatch/keys/computed.go

```diff
@@ -94,5 +94,7 @@ func lookupSubjectsRequestToKey(req *v1.DispatchLookupSubjectsRequest, option di
 		hashableRelationReference{req.ResourceRelation},
 		hashableRelationReference{req.SubjectRelation},
 		hashableIds(req.ResourceIds),
+		hashableCursor{req.OptionalCursor},
```

Comment:
shouldn't we also consider the wildcard option as part of the dispatch cache key, or else 2 requests with different arguments would collide?

Reply:
Yes, good catch.